cypress/e2e/dev/6-management-tests/install-available-plugin.cypress.js

```diff
@@ -2,7 +2,7 @@ const { waitForApplication, uninstallPlugin, installPlugin } = require('../../ut
 const managePluginsPagePath = '/manage-prototype/plugins'
 const plugin = '@govuk-prototype-kit/step-by-step'
 const version1 = '@1.0.0'
-const version2 = '@2.0.0'
+const version2 = '@2.1.0'
```

Comment:
Might be worth allowing patch versions through without needing another PR

```suggestion
const version2 = '@^2.1.0'
```

Reply:
The test is looking for a string.  I don't think this will work in the cypress test.